app/redux/reducers/misc.js

```diff
@@ -1044,6 +1044,7 @@ export const tideDashboardPatients = (state = initialState.tideDashboardPatients
     case types.FETCH_TIDE_DASHBOARD_PATIENTS_SUCCESS:
       return action?.payload?.results || initialState.tideDashboardPatients;
     case types.LOGOUT_REQUEST:
+    case types.CLEAR_TIDE_DASHBOARD_PATIENTS:
```

Comment:
I think this could use a unit test (more of a guarantee of functionality into the future than a "does this do what we want")

Reply:
Oh, right - sorry; my mind was in Uploader where the reducer unit tests are under `misc.test.js` instead of individual files per reducer and I was just quickly trying to match up file-for-file unit tests at the end of the review (I had already hit the "viewed" checkbox on that unit test, so out-of-sight/out-of-mind took over). All good 👍🏼